### ISSUE TITLE
Header OSLC-Core-Version hard coded in OslcQuery, I cannot pass version 3.0 in header

### ISSUE THREAD

**Issue by dkedarptc** [eclipse/lyo.client#68]
Response getResponse() {

		Response cr = null;
		cr = queryResource.request(OSLCConstants.CT_RDF)
				.header(OSLCConstants.OSLC_CORE_VERSION,"2.0")
				.get();

		return cr;
	}

**Comment by berezovskyi** [comment]
You are right, @dkedarptc. I have added a new constructor parameter to both existing client constructors and also added a few constants for the existing and planned OSLC Core/Domain versions to `OSLCConstants`.

Please reopen the issue if something is still blocking you from customising the version.

cc @jamsden @jadelkhoury

**Comment by berezovskyi** [comment]
Docs: https://download.eclipse.org/lyo/docs/oslc4j-client/latest/org/eclipse/lyo/oslc4j/client/OSLCConstants.html#OSLC2_0 and https://download.eclipse.org/lyo/docs/oslc4j-client/latest/org/eclipse/lyo/oslc4j/client/OslcClient.html#OslcClient-java.lang.String-

**Comment by dkedarptc** [comment]
I understand you made changes on same line. But, I still not understand, the hard coding of oslc-core-version=2.0 in OslcQuery->createresource

private Resource createQueryResource() { 
 		Resource resource = oslcClient.getQueryResource(this); 
 		resource.accept(OSLCConstants.CT_RDF); 
 		resource.header(OSLCConstants.OSLC_CORE_VERSION,"2.0"); 
 		applyPagination(resource); 
		applyOslcQueryParams(resource); 
 		return resource; 
} 

Basically, in OSLC client I am able to pass OSLC_CORE_VERSION as 3.0, but while calling OslcQuery it is internally passing as 2.0. Due to this somehow my integration is failing as I am restricting header as 3.0 only.

**Comment by berezovskyi** [comment]
Now I got it, @dkedarptc! I have added an extra constructor to the OslcQuery class.  Please try it out and tell me how it goes.

**Comment by berezovskyi** [comment]
https://download.eclipse.org/lyo/docs/oslc4j-client/latest/org/eclipse/lyo/oslc4j/client/resources/OslcQuery.html#OslcQuery-org.eclipse.lyo.oslc4j.client.OslcClient-java.lang.String-int-org.eclipse.lyo.oslc4j.client.resources.OslcQueryParameters-java.lang.String-

**Comment by dkedarptc** [comment]
It is working. Thanks for your help !
